addons/mail/static/src/messaging/component/messaging_menu/messaging_menu.xml

```diff
@@ -2,30 +2,30 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.messaging.component.MessagingMenu" owl="1">
-        <li class="o_MessagingMenu" t-att-class="{ show: messagingMenu.isOpen, 'o-mobile': storeProps.isMobile }">
+        <li class="o_MessagingMenu" t-att-class="{ show: messagingMenu.isOpen, 'o-mobile': env.entities.Device.instance.isMobile }">
             <!-- TODO FIXME restore mobile stuff here (activeMobileThread) -->
-            <t t-if="!storeProps.isMobile or true">
+            <t t-if="!env.entities.Device.instance.isMobile or true">
                 <a class="o_MessagingMenu_toggler o-no-caret" t-att-class="{ show: messagingMenu.isOpen }" href="#" t-on-click="_onClickToggler" data-toggle="dropdown" data-display="static" title="Conversations" role="button" aria-expanded="false">
                     <i class="o_MessagingMenu_icon fa fa-comments-o" role="img" aria-label="Messages"/>
-                    <t t-if="!storeProps.isMessagingInitialized">
+                    <t t-if="!env.entities.Messaging.instance.isInitialized">
                         <i class="o_MessagingMenu_loading fa fa-spinner fa-spin"/>
                     </t>
-                    <t t-elif="storeProps.counter > 0">
+                    <t t-elif="messagingMenu.counter > 0">
```

Comment:
getter not used in prep? not possible here?

Reply:
no because computation is made by component in prep, while it's done in entity on this branch